mlrun/runtimes/pod.py

```diff
@@ -337,7 +337,10 @@ def _resolve_if_type_sanitized(attribute_name, attribute):
 
     def _set_volume_mount(self, volume_mount):
         # using the mountPath as the key cause it must be unique (k8s limitation)
-        self._volume_mounts[get_item_name(volume_mount, "mountPath")] = volume_mount
+        self._volume_mounts[
+            get_item_name(volume_mount, "mountPath")
+            or get_item_name(volume_mount, "mount_path")
```

Comment:
what is the scenario in which we will have `mount_path`?

Reply:
Added a comment